modules/custom_operations/user_ie_extensions/tokenizer/sentence_piece.cpp

```diff
@@ -157,24 +134,14 @@ bool SentencepieceTokenizer::evaluate(TensorVector& outputs, const TensorVector&
     if(input_element_type == ov::element::string) {
```

Comment:
Please remove artifacts of packed strings: `begin_ids`, etc. above

Reply:
Deleted.